CHANGELOG.md

```diff
@@ -32,6 +33,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Improve type resolution on binary and unary expressions.
 - Improve type comparisons between signed and unsigned integer types.
+- Improve type comparisons between text types.
+- Improve type conversions from character pointers to strings.
```

Comment:
This doesn't sound quite right to me. Maybe "Improve accuracy of" instead of just "Improve"?